.gitbook/5-oracles/1-operate-oracle-nodes/3-genesis-oracle.md

```diff
@@ -0,0 +1,212 @@
+|-----------------------------------|-------------|------------------------------------------------------------------------------------------------------|
+| oracle-unique-id                  | false       | The uniqueID to be set in the params of the oracle module and the genesis oracle                     |
+| oracle-account                    | false       | The address or key name of the account to be registered as an genesis oracle                         |
+| oracle-commission-rate            | false       | The desired initial oracle commission rate                                                           |
+| oracle-commission-max-rate        | false       | The maximum oracle commission rate. The oracle commission rate cannot be greater than this max rate. |
+| oracle-commission-max-change-rate | false       | The maximum rate that an oracle can change once. It will be reset 24 hours after the last change.    |
+| oracle-endpoint                   | false       | The endpoint of oracle to be used                                                                    |
+
+You can check the oracle registered in `genesis.json`
+```
+cat $HOME/.panacea/config/genesis.json | jq .app_state.oracle.oracles
+```
+**Output**
+```json
+[
+  {
+    "oracle_address": "{oracle_address}",
+    "unique_id": "{unique_id}",
+    "endpoint": "{endpoint}",
+    "update_time": "0001-01-01T00:00:00Z",
+    "oracle_commission_rate": "{commission_rate}",
+    "oracle_commission_max_rate": "{commission_max_rate}",
+    "oracle_commission_max_change_rate": "{commission_max_change_rate}"
+  }
+]
+```
+
+### Start block generation in Panacea
+You need to launch the Panacea to start generating blocks. 
+```shell
+panacead start
+```
+
+## Registering Oracle keys and Remote Report
+
+The genesis oracle must create an oracle private key and public key to use for data encryption/decryption.
+It also issues a remote report to prove that it is a valid oracle.
+
+### Get trusted block information
+In order to generate oracle keys and remote reports, the oracle need trusted block information first. This block information is used by the light client to validate the data retrieved from Panacea.
+
+In fact, Oracle doesn't need this block information because it doesn't retrieve Panacea's data when generating oracle keys and remote reports. 
+However, when the oracle participates in the verification operation (`oracled start`), the oracle needs to use a light client as it will retrieve data from Panacea.
+Therefore, unless reliable block information is received during the process of generating an oracle key, the genesis oracle has no way to retrieve this block information.
+
+You can get trusted block information by:
+```shell
+BLOCK=$(panacead q block --node <node-rpc-address>)
+
+HEIGHT=$(echo $BLOCK | jq -r .block.header.height)
+HASH=$(echo $BLOCK | jq -r .block_id.hash)
+```
```

Comment:
I think this block can be put inside the below section, `Generates oracle's private key, ...`
The flow I think is,

1. let's generate oracle key
2. but for `gen-oracle-key` CLI, trusted block infromation is required. because..
3. guide to get trusted block info
4. `gen-oracle-key --trusted-block-height...`

wdyt?

Reply:
Yes, I think it looks good.
I will change it as your said.